app/inbound/save-invoice-lines.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../data')
+
+const saveInvoiceLines = async (invoiceLines, paymentRequestId, transaction) => {
+  for (const invoiceLine of invoiceLines) {
+    delete invoiceLine.invoiceLineId
+    invoiceLine.fundingCode = invoiceLine.schemeCode
+    delete invoiceLine.schemeCode
+    await db.invoiceLine.create({ paymentRequestId, ...invoiceLine }, { transaction })
```

Comment:
I'd reverse these arguments otherwise if `invoiceLine` already has a `paymentRequestId` you'll overwrite the one you've specifically set in.